metricbeat/docs/modules/ceph.asciidoc

```diff
@@ -10,6 +10,11 @@ beta[]
 The Ceph module collects metrics by submitting HTTP GET requests to
 the http://docs.ceph.com/docs/master/man/8/ceph-rest-api/[ceph-rest-api]. The default metricsets are `cluster_disk`, `cluster_health`, `monitor_health`, `pool_disk`, `osd_tree`.
 
+[float]
+=== Compatibility
+
+The Ceph module is tested with Ceph Jewel
```

Comment:
Could you add a dot at the end? I think also applies to more entries below. Should we also put a version number here? (10.2)?

Reply:
Added, added version number as in the docker image we use too.